handbook/company/head-of-demand-generation.md

```diff
@@ -0,0 +1,65 @@
+# Head of Demand Generation
+
+## Let's start with why we exist. 📡
+
+Ever wondered if your employer is monitoring your work computer?
+
+Organizations make huge investments every year to keep their laptops and servers online, secure, compliant, and usable from anywhere. This is called "device management".
+
+At Fleet, we think it's time device management became [transparent](https://fleetdm.com/transparency) and [open source](https://fleetdm.com/handbook/company#open-source).
+
+
+## About the company 🌈
+
+You can read more about the company in our [handbook](https://fleetdm.com/handbook/company), which is public and open to the world.
+
+tldr; Fleet Device Management Inc. is a [recently-funded](https://techcrunch.com/2022/04/28/fleet-nabs-20m-to-enable-enterprises-to-manage-their-devices/) Series A startup founded and backed by the same people who created osquery, the leading open source security agent. Today, osquery is installed on millions of laptops and servers, and it is especially popular with [enterprise IT and security teams](https://www.linuxfoundation.org/press/press-release/the-linux-foundation-announces-intent-to-form-new-foundation-to-support-osquery-community).
+
+### Your primary responsibilities 🔭
+
+As Head of Demand Generation at Fleet, you will get the chance to…
+
+In your first 90 days:
+
+- 🔗 Collaborate closely with the CEO and Head of Sales on all Demand Gen efforts.
+- 🗺️ Lead and manage workflow for team contributors, particularly Fleet's existing field marketer and community+content marketer.
+- 🥼 Learn and invest in the existing ABM and enterprise strategy by expanding it through rigorous experimentation.
+- 🎥 Manage content marketing, community marketing, and enterprise events.
+- ⚡️ Build and evolve strategies and systems around tightly coordinating physical events and digital events.
+
+### Are you our new team member? 🧑‍🚀
+
+If most of these qualities sound like you, we would love to chat and see if we're a good fit.
+
+### Experience
+
+- 🧙‍♂️ Wizardy abilities in demand generation campaigns
+  - ⛰️ Recent real-world GA4 experience in an enterprise setting.
+  - 👔 LinkedIn ads
+- Strong opinions on Google and LinkedIn adds with recent (≥2020) hands-on experience.
+- 🤘 Hands-on marketing operator and capability to manage tools expertly, personally by hand 
+- 🎙️ Experience with open source and how to talk to technical people about it.
+- 🧗‍♂️ Experience marketing to enterprise accounts; ideally bottom-up.
+
+
+### You can "walk the walk":
+
+- ⭐️ An up-and-coming leader or someone with no ego who will get their hands dirty.
+- 🤔 Systems thinker. Process-oriented. Analytical.
+- 🤝 Collaboration: You work best in a participatory, team-based environment.
+- 🧑‍🏭 Prototype-first: You embrace speed and failure as we iterate towards the right solution. 
+- 🧬 Simplicity: You love complex questions and use your work to simplify that complexity.
```

Comment:
cut off "for users."

Needs to be updated in hard-coded string: https://github.com/fleetdm/fleet/commit/2708933529338c2219af39275d94bb0b9e10d674